### ISSUE TITLE
Please add a function to Color such as Color.FromHDR(Long) for Visual Basic and Visual C# 2022.

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/runtime#78403]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Please-add-a-function-to-Color-such-as-C/1485038)._

---
No problem, just a simple request. I have a HDR display, a HDR Video Cable and a HDR enabled computer.

---
#### Feedback Bot on 7/22/2021, 10:51 PM: 

(private comment, text removed)

**Comment by terrajobst** [comment]
@OliaG do you know how well WinForms/WPF support HDR to begin with?

**Comment by OliaG** [comment]
> @OliaG do you know how well WinForms/WPF support HDR to begin with?

I’ll divert to @merriemcgaw

**Comment by JeremyKuhne** [comment]
I don't believe there is any support in GDI+, but this is about converting from an HDR color, not using them directly I believe?

@tannergooding do you think this is better handled in the new Color structs? In any case, I'd want to see more details about a specific color format involved and the end-to-end scenario.

**Comment by teo-tsirpanis** [comment]
Can this be moved to `dotnet/winforms`?
